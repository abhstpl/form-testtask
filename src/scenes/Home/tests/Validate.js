import React from 'react';
import validate from '../validate';

describe('Test on validate', () => {
    it('without Values', () => {
        const values = {
        };
        validate(values);
        expect(validate).toMatchSnapshot();
    });
    it('with values', () => {
        const values = {
            firstName: 'testtesttesttesttest',
            email: 'test@test.com'
        };
        validate(values);
        expect(validate).toMatchSnapshot();
    });
    it('with wrong values', () => {
        const values = {
            firstName: 'test',
            email: 'test'
        };
        validate(values);
        expect(validate).toMatchSnapshot();
    });
});