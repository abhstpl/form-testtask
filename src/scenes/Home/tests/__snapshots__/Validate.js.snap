// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on validate with values 1`] = `[Function]`;

exports[`Test on validate with wrong values 1`] = `[Function]`;

exports[`Test on validate without Values 1`] = `[Function]`;
